qa/tasks/scripts/ses_network_failure.sh

```diff
@@ -0,0 +1,245 @@
+set -ex
+
+pool_name=$(ceph osd pool ls)
+rbd -p $pool_name create image1 --size 2G
+rbd_device=$(rbd map $pool_name/image1)
+parted -s $rbd_device mklabel gpt unit % mkpart 1 xfs 0 100
+mkfs.xfs ${rbd_device}p1
+mount ${rbd_device}p1 /mnt
+
+nettest () {
+set +x
+time_consumed+="$1 - $( (time dd if=/dev/zero  of=/mnt/file1.bin count=100 bs=1M oflag=direct >/dev/null 2>&1) 2>&1 | sed '/^$/d' | head -1); "
+set -x
+ls -l /mnt/file1.bin
+rm -f /mnt/file1.bin
+}
+
+create_netem_rules () {
```

Comment:
this is a function for readability add 'function' keyword, and in the body add indentation.

Reply:
...and for self-documented code, could you please add local variables for arguments.

Reply:
not everywhere